apps/src/applab/designMode.js

```diff
@@ -496,22 +499,29 @@ function makeDraggable (jq) {
 
 /**
  * Inverse of `makeDraggable`.
- * @param {jQuery} jq jQuery object containing DOM elements to make undraggable.
+ * @param {jQuery} jqueryElements jQuery object containing DOM elements to make
+ *   undraggable.
+ * @returns {boolean} True if we made something undraggable
  */
-function makeUndraggable(jq) {
-  jq.each(function () {
+function makeUndraggable(jqueryElements) {
+  var foundOne = false;
+  jqueryElements.each(function () {
     var wrapper = $(this);
     var elm = $(getInnerElement(this));
 
     // Don't unwrap elements that aren't wrapped with a draggable div.
-    if (!wrapper.data('uiDraggable')) {
+    if (!wrapper.hasClass('ui-draggable')) {
```

Comment:
wrapper.data('uiDraggable') wasn't working in phantom. This left me worried it may not work in some other browsers elsewhere, so I'm just going to use hasClass, which works fine.